index.js

```diff
@@ -1,8 +1,121 @@
-const bulk = require('bulk-require')
 
 module.exports = {
-  patchcore: bulk(__dirname, [
-    './+(config|emoji|invite|keys|sbot).js',
-    './+(about|backlinks|blob|channel|contact|feed|lib|message|router)/**/*.js'
-  ])
+  patchcore: {
+    config: require('./config'),
+    emoji: require('./emoji'),
+    invite: require('./invite'),
+    keys: require('./keys'),
+    sbot: require('./sbot'),
+    about: {
+      obs: require('./about/obs'),
+      sync: require('./about/sync'),
+      html: {
+        image: require('./about/html/image'),
+        link: require('./about/html/link'),
+      },
+    },
+    backlinks: {
+      obs: require('./backlinks/obs')
```

Comment:
I recently added /backlinks/obs-filter.js . Would you mind including that too? =]

Reply:
added... although, that doesn't follow the naming convention (I disagree with the convention, but matters of principle! anyway I added this as it lies, not gonna fix to follow the convention in this PR)

Reply:
I followed matt's advice on the naming of the module: https://github.com/ssbc/patchcore/pull/46/files/b70e39416e0ec7a808742b9841a6cd32d45b3354#r153082169

And then Matt renamed the file from `obs_filter` to `obs-filter`: https://github.com/ssbc/patchcore/commit/81c203a535afa4c5298f1d7c11c559e89d016355

Is it the filename rather than the module name I got wrong? Should the filename be `filter.js` to follow the convention? (Or am I misreading the thread on scuttlebutt)? :P

Reply:
if there is more than one file, there should be an obs directory
```
backlinks/
  obs/
    filter.js
    for.js //previously named obs.js
```
(I am not a fan of this convention, but I am a fan of following your conventions consistently!)

Reply:
Noted. I'll submit a PR moving it when I next have time =].

Cheers.